charts/fybrik/files/taxonomy/data_catalog_response.json

```diff
@@ -0,0 +1,76 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Data Catalog Response - GetAssetDetails",
+  "type": "object",
+  "properties": {
+    "name": {
+      "$ref": "#/definitions/assetName"
+    },
+    "asset": {
+      "$ref": "#/definitions/assetSpec"
+    }
+  },
+  "required": [
+    "name",
+    "asset"
+  ],
+  "additionalProperties": false,
+  "definitions": {
+    "assetName": {
+      "type": "string",
+      "description": "Name of the asset"
+    },
+    "assetSpec": {
+      "type": "object",
+      "properties": {
+        "assetCredentials": {
+          "$ref": "#/definitions/vaultPluginPath"
+        },
+        "assetMetadata": {
+          "$ref": "#/definitions/assetMetadata"
+        },
+        "assetDetails": {
+          "$ref": "#/definitions/assetDetails"
+        }
+      },
+      "required": [
+        "assetCredentials",
+        "assetMetadata",
+        "assetDetails"
+      ]
+    },
+    "vaultPluginPath": {
+      "type": "string",
+      "description": "This has the vault plugin path where the data credentials will be stored as kubernetes secrets. This value is assumed to be known to the catalog connector."
+    },
+    "assetDetails": {
+      "type": "object",
+      "description": "Asset details",
+      "properties": {
+        "connection": {
+          "$ref": "taxonomy.json#/definitions/Connection"
+        },
+        "dataFormat": {
+          "$ref": "taxonomy.json#/definitions/Dataformat"
+        }
+      },
+      "required": [
+        "connection"
+      ]
+    },
+    "assetMetadata": {
+      "type": "object",
+      "properties": {
+        "owner": {
+          "type": "string"
+        },
+        "geography": {
+          "type": "string"
+        },
+        "metadata": {
```

Comment:
I find it confusing to have a `metadata` field inside `assetMetadata`. It might be better to add `owner` and `geography` directly to `data_catalog.json#/definitions/Resource` instead of wrapping (they might be valuable in the policy manager context as well).

Overall it seems like `data_catalog.json#/definitions/Resource` can represent `ResourceMetadata` and be a top level property of the response object and its only sibling is the details property with connection information and format.

Reply:
Changed the structure of the data catalog response.json based on the above comments. Please review and let me know if there are any changes needed. Thanks.